draft-ietf-quic-transport.md

```diff
@@ -1641,6 +1641,14 @@ of connection establishment.  By giving the client a different connection ID to
 use, a server can cause the connection to be routed to a server instance with
 more resources available for new connections.
 
+If a server receives a client Initial with a unverifiable Retry token,
```

Comment:
Maybe s/unverifiable/invalid/?

To me it seems that "unverifiable" might have the implication of the decryption process (or the integrity check) of the token failing. But a rejection of Retry token happens in other cases too, e.g., when the Retry token is outdated.

Reply:
Done.